installing/installing_rhv/installing-rhv-creating-custom-vm.adoc

```diff
@@ -0,0 +1,53 @@
+[id="installing-rhv-creating-custom-vm"]
+= Creating a custom virtual machine template on {rh-virtualization}
+include::modules/common-attributes.adoc[]
```

Comment:
Common attributes need to come before you use something from those attributes. Since this is the first file in the overall install_rhv book it shows the problem immediately (on portal), but the issue is not obvious for the following assemblies. 

Fixed via: https://github.com/openshift/openshift-docs/pull/21666

Reply:
Yikes, looks like that in itself broke something else. To fix this for GA, I will put this back and hardcode the RHV substitution. Will need to figure out what went wrong here later.